cmd/syft/internal/options/source.go

```diff
@@ -32,6 +34,7 @@ var _ interface {
 var _ clio.PostLoader = (*imageSource)(nil)
 
 func (o *sourceConfig) DescribeFields(descriptions clio.FieldDescriptionSet) {
+	descriptions.Add(&o.Supplier, `optional field for syft user to configure the supplier of the root software component being analyzed`)
```

Comment:
FYI This doesn't need a description here because it has a flag bound to this value, the description should be picked up from the flag description.